submissions/opeyemiPeter/css/product.css

```diff
@@ -0,0 +1,309 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Montserrat;
+  font-size: 18px;
+  line-height: 22px;
+  overflow-x: hidden;
+}
+a {
+  text-decoration: none;
+}
+
+/********************
+        header
+**********************/
+header {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: flex-end;
+  background: #1b1a1b;
+  align-items: center;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+.logo {
+  margin-right:auto;
+  margin-left: 75px;
+}
+.logo a {
+  font-family: "", cursive;
+  color: #1b82d6;
+  font-weight: 100; 
+}
+.span-logo {
+  color: #1b82d6;
+  font-family: "Grenze Gotisch", cursive;
```

Comment:
You already set the font family in the body tag. No need for it here. @opeyemipeter1759

Reply:
@ToluAkin the font-family is for span in the logo, it was intended to be different from that of the rest, reasons why i declared it there.

Reply:
@ToluAkin good morning, sequel to your comment on my PR, the font family i wanted for the logo was suppose to be entirely different from that of the rest, reasons why i declared a different font family for the logo.